### DISCUSSION TITLE
How to remove slider in umijs

### DISCUSSION THREAD

**Discussion by Khadhiri786** [umijs/umi#11962] [General]
I have created umijs project, how to remove slider in this project
 `import { defineConfig } from '@umijs/max';

export default defineConfig({
  antd: {},
  access: {},
  model: {},
  initialState: {},
  request: {},
  layout: {
    name: 'Test',
    locale: false,
  },

  routes: [
    {
      path: '/',
      redirect: '/home',
    },
    {
      name: '首页',
      path: '/home',
      component: '@/layouts/BasicLayout',
      
    },
    {
      name: '权限演示',
      path: '/access',
      component: './Access',
    },
    {
      name: ' CRUD 示例',
      path: '/table',
      component: './Table',
      
    },
  ],
  npmClient: 'yarn',
});

`

**Comment by fz6m** [comment]
config `export const layout` (in `src/app.tsx`) to change the behavior of the layout :

```ts
// src/app.tsx

export const layout = () => {
  return {
    // Refer: https://procomponents.ant.design/components/layout
    // options
  }
}
```
